flow/layers/backdrop_filter_layer_unittests.cc

```diff
@@ -481,6 +481,31 @@ TEST_F(BackdropLayerDiffTest, BackdropLayer) {
   EXPECT_EQ(damage.frame_damage, SkIRect::MakeLTRB(0, 0, 190, 190));
 }
 
+TEST_F(BackdropLayerDiffTest, ReadbackOutsideOfPaintArea) {
+  auto filter = DlMatrixImageFilter(SkMatrix::Translate(50, 50),
+                                    DlImageSampling::kLinear);
+
+  MockLayerTree l1(SkISize::Make(100, 100));
+
+  auto clip = std::make_shared<ClipRectLayer>(SkRect::MakeLTRB(60, 60, 80, 80),
+                                              Clip::hardEdge);
+  clip->Add(std::make_shared<BackdropFilterLayer>(filter.shared(),
+                                                  DlBlendMode::kSrcOver));
+  l1.root()->Add(clip);
+  auto damage = DiffLayerTree(l1, MockLayerTree(SkISize::Make(100, 100)));
+
+  EXPECT_EQ(damage.frame_damage, SkIRect::MakeLTRB(10, 10, 80, 80));
```

Comment:
What do you think about writing the tests like this?
```suggestion
  EXPECT_EQ(damage.frame_damage, SkIRect::MakeLTRB(clip_rect.origin.x - translation.x, 10, 80, 80));
```

Reply:
I thought the general consensus was to not calculate the result values in test from the input values. Maybe that's not the case, or I took it too literally?

Reply:
Having just read through the previous test it definitely would have helped me.  Some middle ground would be to use `60 - 50` instead of `10`.  I think the concern is someone calculating the assertion with the same code as they are checking?